src/main/play-doc/operation/ContinuousDeliverySetupOps.md

```diff
@@ -0,0 +1,134 @@
+# Continuous Delivery
+
+Continuous Delivery is a ConductR feature which allows bundles to deployed to a ConductR cluster upon being published to Bintray.
+
+From the perspective of Continuous Delivery setup, the ops are responsible for installation of the Continuous Delivery service of ConductR as well as configuration of Bintray webhook which communicates with Continuous Delivery service.
+
+## Requirements
+
+Continuous Delivery functionality requires a working ConductR Cluster with ConductR HAProxy deployed. ConductR HAProxy is required to allow Bintray webhook access to Continuous Delivery service. ConductR HAProxy installation instructions is available on the [[Install|Install]] page.
+
+Continuous Delivery functionality requires Bintray webhook to be setup. As such, Bintray credentials with publish permission and package read/write entitlement is required.
+
+## Bintray webhook secret
+
+The webhook secret is the Bintray API key of the credentials owning the Bintray webhook.
+
+For the purpose of this documentation, we will be using `bt-secret` in place of the actual API key value.
+
+## Installing Continuous Delivery service
+
+The Continous Delivery service needs to be configured with the webhook secret. Prepare the bundle configuration as such, replacing `bt-secret` with the actual Bintray API key value.
+
+```
+$ mkdir /tmp/cd-config
+$ echo 'export BINTRAY_WEBHOOK_SECRET=bt-secret' > /tmp/cd-config/runtime.config.sh
+$ shazar /tmp/cd-config
+```
+
+The bundle configuration for the Bintray webhook secret will be generated by running the commands above.
+
+Load the Continuous Delivery bundle with the generated bundle configuration and run the service.
+
+```
+conduct load continuous-delivery <path to generated bundle configuration>
```

Comment:
Should you not substitute the `/tmp/cd-config` path that you used above instead?